packages/api/src/auth/auth.service.ts

```diff
@@ -1,153 +1,107 @@
+      let newTokens: Tokens;
+      const user = await this.usersService.findOne(refreshPayload.sub);
+      if (user) {
+        newTokens = this.generateTokens(user, refreshPayload.keepMeLogin);
+        // TODO save refresh token in db
+      } else {
+        throw new Error('Invalid subscriber ID');
+      }
+      return newTokens;
+    } catch (err) {
+      throw new UnauthorizedException('Invalid tokens');
     }
-
-    return this.usersService.findOneByEmail(this.tokenContent.email);
   }
 
-  async login({ email, id, roles }: UserEntity) {
-    return {
-      access_token: this.jwtService.sign({ email, sub: id, roles }),
-    };
+  public async getUser(): Promise<Promise<UserEntity> | undefined> {
+    return undefined;
   }
 
-  async register(user: CreateUserDto) {
-    const createdUser = await this.usersService.create(user);
-    const { email, roles, id } = createdUser;
-    return {
-      access_token: this.jwtService.sign({ email, sub: id, roles }),
-    };
+  public async logout(refreshToken: string) {
+    // TODO remove token
   }
 
-  private async decodeAndVerifyToken(token: string): Promise<TokenPayload> {
-    if (this.tokenContent) {
-      return this.tokenContent;
-    }
-
-    const tokenContent: unknown = jsonwebtoken.decode(token, {
-      complete: true,
-    });
-
-    if (!AuthService.isTokenContentValid(tokenContent)) {
-      throw new Error('Invalid token content');
-    }
-
-    const tokenPublicKey = tokenContent.header.kid;
-    const verifiedTokenContent: TokenPayload = jsonwebtoken.verify(
-      token,
-      tokenPublicKey,
-    ) as TokenPayload;
-
-    this.tokenContent = verifiedTokenContent;
-    return verifiedTokenContent;
-  }
-
-  private static isTokenContentValid(tokenContent: unknown): tokenContent is CognitoTokenContent {
-    return Boolean(tokenContent) && typeof tokenContent !== 'string';
-  }
-
-  private extractTokenFromHeaders(): string {
-    const authorizationHeader: string | undefined = this.request.headers.authorization;
-    if (!authorizationHeader) {
-      throw new Error('Missing Authorization header');
-    }
-
-    const [bearer, token] = authorizationHeader.split(' ');
-
-    if (bearer !== 'Bearer' || !token) {
-      throw new Error('Bad Authorization header');
-    }
-
-    return token;
+  async register(user: CreateUserDto): Promise<UserEntity> {
+    return this.usersService.create(user);
   }
 
-  private verifyRole(requiredRole: Role): void {
-    if (!this.tokenContent) {
-      throw new UnauthorizedException();
-    }
-
-    const groupFromToken: Role[] | undefined = this.tokenContent.group;
-
-    if (!groupFromToken || !groupFromToken.includes(requiredRole)) {
-      throw new ForbiddenException();
-    }
+  async validateUser(email: string, passwordHash: string): Promise<UserEntity> {
+    const user = await this.usersService.findOneByEmail(email);
+    if (user.password !== passwordHash) throw new NotFoundException('User password mismatch');
```

Comment:
Please use always brackets. It also works the same but to keep consistency and standard we want to always use brackets.